auth-web/src/services/business.services.ts

```diff
@@ -152,6 +167,27 @@ export default class BusinessService {
     // return axios.get(url)
   }
 
+  /**
+   * Saves a continuation review result.
+   * @param reviewResult the review result object
```

Comment:
Wrong comment here? I see two params and none of them is the `reviewResult`.

Reply:
Good catch. Yes, I refactored the method parameters and forgot to update the JS Doc.

Fixed in next commit.